src/features/profile/Profile.tsx

```diff
@@ -0,0 +1,94 @@
+import { useCallback, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import { useSelector } from "react-redux";
+import useProfileState from "./use-profile-state";
+import { AuthState } from "features/auth/store/types";
+import { fetchOneProfile } from "./services/profile.service";
+import ShowSkills from "./components/show-skills";
+import ShowExperiences from "./components/show-experiences";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import IconButton from "@mui/material/IconButton";
+import EditIcon from "@mui/icons-material/Edit";
+
+const Profile = () => {
+  const { profile, setProfile } = useProfileState();
+
+  const id = parseInt(useParams<RouteParams>().id!);
+
+  function useAuthState(): AuthState {
+    return useSelector<{ auth: AuthState }>((state) => state.auth) as AuthState;
+  }
+  const { token } = useAuthState();
+  const loggedUser = useAuthState().user;
+
+  const getProfile = useCallback(async () => {
+    try {
+      if (token && id) {
+        const data = await fetchOneProfile(token, id);
+        setProfile(data);
+        console.log(token);
+      }
+    } catch (error) {
+      console.log(error);
```

Comment:
Deberiamos evitar que lleguen console.log a produccion
```suggestion
      console.log(error);
```
Aunque en este caso como es un error podria convenir... pero idk
```suggestion
      console.error(error);
```
Pero muchisisimo mas importante, donde esta la recuperacion o la notificacion al usuario de que hubo un error 🧐??
```suggestion
      mostrarNotificacionDeError(error);
```
toastify??

Reply:
enfocaria eso para el siguiente sprint, son detalles que todavia no he podido investigar, y no son prioridad por ahora

Reply:
Con el siguiente Sprint te refieres a este verdad xd?, para el anterior si es medio equisde, pero para ahorita si lo veo prioritario xP